holoviews/plotting/bokeh/__init__.py

```diff
@@ -33,6 +33,9 @@
 
 Store.renderers['bokeh'] = BokehRenderer.instance()
 
+if len(Store.renderers) == 1:
```

Comment:
Yes, ``player`` does not use ``BokehRenderer.app`` it's an example of creating a custom bokeh layout with custom widgets.